documentation/docs/05-load.md

```diff
@@ -256,7 +256,7 @@ export async function load({ setHeaders }) {
 
 ### Output
 
-Any promises on the returned `data` object will be resolved, if they are top-level properties. This makes it easy to return multiple promises without creating a waterfall:
+On the client, the only requirement for the return value is to be a dictionary at the top level. On the server, the output also needs to be JSON-serializable. On both the client and server any promises on the returned `data` object will be resolved, if they are top-level properties. This makes it easy to return multiple promises without creating a waterfall:
```

Comment:
I think the output stuff should go _after_ the promise discussion, because it's weird to say 'it has to be JSON-serializable' immediately followed by 'the top-level properties can be promises'. I think we could change this sentence back, and specify the constraints after the code sample

```suggestion
Any promises on the returned `data` object will be resolved, if they are top-level properties. This makes it easy to return multiple promises without creating a waterfall:
```

Reply:
Yeah that's better.